components/MemberList/index.js

```diff
@@ -5,34 +5,39 @@ import Input from '../Input';
 
 const Member = ({ member }) => {
   return (
-    <td className="px-6 py-4 whitespace-nowrap">
-      <Link href={'/members/' + member.id}>
-        <a>
-          <div className="flex items-center">
-            <div className="flex-shrink-0 h-10 w-10">
-              {/*eslint-disable-next-line*/}
-              <img
-                className="h-10 w-10 rounded-full"
-                src={member.image}
-                alt=""
-              />
-            </div>
-            <div className="ml-4">
-              <div className="text-sm font-medium text-gray-900">
-                {member.name}
+    <>
+      <td className="px-6 py-4 whitespace-nowrap">
+        <Link href={'/members/' + member.id}>
+          <a>
+            <div className="flex items-center">
+              <div className="flex-shrink-0 h-10 w-10">
+                {/*eslint-disable-next-line*/}
+                <img
+                  className="h-10 w-10 rounded-full"
+                  src={member.image}
+                  alt=""
+                />
+              </div>
+              <div className="ml-4">
+                <div className="text-sm font-medium text-gray-900">
+                  {member.name}
+                </div>
+                <div className="text-sm text-gray-500">{member.email}</div>
               </div>
-              <div className="text-sm text-gray-500">{member.email}</div>
             </div>
-          </div>
-        </a>
-      </Link>
-    </td>
+          </a>
+        </Link>
+      </td>
+      <td className="px-6 py-4 whitespace-nowrap">
```

Comment:
we can make a TD component in this file and use it as `<TD></TD>` so we do not have to repeat the styling!